AdyenComponents/Apple Pay/ApplePayComponentExtensions.swift

```diff
@@ -24,7 +24,6 @@ extension ApplePayComponent: PKPaymentAuthorizationViewControllerDelegate {
     public func paymentAuthorizationViewController(_ controller: PKPaymentAuthorizationViewController,
                                                    didAuthorizePayment payment: PKPayment,
                                                    completion: @escaping (PKPaymentAuthorizationStatus) -> Void) {
-        paymentAuthorizationCompletion = completion
```

Comment:
I don't think this is correct this closure has to be retained and called with success or failure